locale/zh_CN.po

```diff
@@ -1027,12 +1027,12 @@ msgstr "无法创建调试报告。"
 #: ../src/common/debugrpt.cpp:228
 #, c-format
 msgid "Failed to remove debug report file \"%s\""
-msgstr "无法删除调试报告文件 \"%s\""
+msgstr "无法删除调试报告文件 \"%s\"。"
 
 #: ../src/common/debugrpt.cpp:240
 #, c-format
 msgid "Failed to clean up debug report directory \"%s\""
-msgstr "无法清除调试报告目录 \"%s\""
+msgstr "无法清除调试报告目录 \"%s\"。"
```

Comment:
nit. no period `.` in the original English text. I would like to remove the `。` to be consistent. Nothing is wrong though. That is probably just my personal preference.